packages/dnote-server/build.sh

```diff
@@ -0,0 +1,45 @@
+TERMUX_PKG_HOMEPAGE=https://www.getdnote.com/
+TERMUX_PKG_DESCRIPTION="dnote server"
```

Comment:
Please change to a more descriptive description

Reply:
Just used the server directory's README description, but changed the word "directory" to "package" that's as official of a description I could find.